htdocs/survey.php

```diff
@@ -38,6 +38,12 @@ class DirectDataEntryMainPage
     var $key;
 
     var $TestName;
+
```

Comment:
why is this the only varialbe that got a cooment ?

Reply:
It was the only one generating an error.

Without it phan was inferring that it's a string, and generating the error:
`htdocs/survey.php:112 PhanTypeComparisonToArray string to array comparison`